Workload-Security/Integration/aws-python-amazon-inspector-integration-v2/templates/c1ws-inspector-findings-template.yaml

```diff
@@ -0,0 +1,702 @@
+
+Parameters:
+  BucketName:
+    Type: String
+    Description: S3 bucket where the csv file report will be uploaded (If you are using an existing bucket then input the name of your bucket if not then choose a unique bucket name)
+  CreateNewBucket:
+    Type: String
+    AllowedValues:
+      - 'Yes'
+      - 'No'
+    Description: >-
+      Are you creating a new bucket for the reports?
+  SnsSubscriptionEmail:
+    Type: String
+    Description: Email address to be used to subscribe to the SNS Topic
+  CloudOneRegion:
+    Type: String
+    Description: Enter your Cloud One Region
+    Default: us-1
+    AllowedValues:
+      - trend-us-1
+      - us-1
+      - in-1
+      - gb-1
+      - jp-1
+      - de-1
+      - au-1
+      - ca-1
+      - sg-1
+  ApiKey:
+    Type: String
+    NoEcho: "true"
+    Description: Please enter your Cloud One API Key
+  CronSchedule:
+    Type: String
+    Description: Schedule the time you would like to run the vulnerabiliy report using the cron format (Default is every wednesday)
+    Default: cron(0 12 ? * WED *)
+
+Conditions:
+  CreateBucket:
+    !Equals [!Ref CreateNewBucket, "Yes"]
+
+Resources:
+  SNSTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      DisplayName: "inspector-findings"
+      TopicName: !Sub "${AWS::StackName}-Inspector-Findings"
+      Subscription:
+        - Protocol: "email"
+          Endpoint: !Ref SnsSubscriptionEmail
+  #   This secret holds the Trend Micro Cloud One API key.
+  ApiKeySecret:
+    Type: AWS::SecretsManager::Secret
+    Properties:
+      Name: "TrendMicro/CloudOne/ApiKey"
+      SecretString: !Sub |
+        {
+          "ApiKey": "${ApiKey}"
+        }
+  Bucket:
+    Type: AWS::S3::Bucket
+    Condition: CreateBucket
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      BucketName: !Ref BucketName
+  LambdaSecretAcessPolicy: 
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Sid: VisualEditor0
+          Effect: Allow
+          Action:
+          - "secretsmanager:GetSecretValue"
+          - "secretsmanager:DescribeSecret"
+          - "sts:AssumeRole"
+          Resource: !Ref ApiKeySecret
+  IpsRuleAutoAssignLambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+        Version: 2012-10-17
+      ManagedPolicyArns:
+          - arn:aws:iam::aws:policy/AmazonVPCCrossAccountNetworkInterfaceOperations
+          - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+          - arn:aws:iam::aws:policy/AmazonInspector2ReadOnlyAccess
+          - arn:aws:iam::aws:policy/CloudWatchLambdaInsightsExecutionRolePolicy
+          - !Ref LambdaSecretAcessPolicy
+      Path: /
+      Policies:
+        - PolicyName: SendCommandPolicy
```

Comment:
This should be converted to a managed policy.

Reply:
Agree with @raphabot, I don't see a point for a managed policy here. In case is just used once.